### ISSUE TITLE
Can't select a project using the keyboard only from the 8.0 Welcome screen

### ISSUE THREAD

**Issue by nosami** [mono/monodevelop#7074]
<img width="545" alt="image" src="https://user-images.githubusercontent.com/667194/52370903-0ddd5180-2a4c-11e9-93a1-1914e21c178a.png">

This is a regression from 7.8 behaviour. Bonus points if Ctrl-N and Ctrl-P can work too for moving up and down the list as well as cursor keys.

**Comment by rodrmoya** [comment]
Pressing TAB should get you to the projects list, although there is no selection/focus indicator, but if you press an arrow key, you should be able to select a project, and then ENTER to open it. Doesn't that work?

If it does, I guess the recent projects list should have a focus ring?
